test/fixtures/track-long-names.json

```diff
@@ -40,6 +40,7 @@
       {
         "_eventName": "Testing in case someone sends us events ",
```

Comment:
@sperand-io I'm not sure I understand your question, but our previous tests were wrong - because they did not check to make sure _appVersion existed in the output. That is why I added it to many of the tests.